src/main/java/org/kohsuke/github/GHPullRequestReviewCommentBuilder.java

```diff
@@ -0,0 +1,118 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+
+// TODO: Auto-generated Javadoc
+
+/**
+ * Builds up a creation of new {@link GHPullRequestReviewComment}.
+ *
+ * @see GHPullRequest#createReviewComment()
+ */
+public class GHPullRequestReviewCommentBuilder {
+    private final GHPullRequest pr;
+    private final Requester builder;
+
+    /**
+     * Instantiates a new GH pull request review comment builder.
+     *
+     * @param pr
+     *            the pr
+     */
+    GHPullRequestReviewCommentBuilder(GHPullRequest pr) {
+        this.pr = pr;
+        this.builder = pr.root().createRequest();
+    }
+
+    /**
+     * The SHA of the commit that needs a review. Not using the latest commit SHA may render your review comment
+     * outdated if a subsequent commit modifies the line you specify as the position. Defaults to the most recent commit
+     * in the pull request when you do not specify a value.
+     *
+     * @param commitId
+     *            the commit id
+     * @return the gh pull request review comment builder
+     */
+    public GHPullRequestReviewCommentBuilder commitId(String commitId) {
+        builder.with("commit_id", commitId);
+        return this;
+    }
+
+    /**
+     * The text of the pull request review comment.
+     *
+     * @param body
+     *            the body
+     * @return the gh pull request review comment builder
+     */
+    public GHPullRequestReviewCommentBuilder body(String body) {
+        builder.with("body", body);
+        return this;
+    }
+
+    /**
+     * The relative path to the file that necessitates a comment.
+     *
+     * @param path
+     *            the path
+     * @return the gh pull request review comment builder
+     */
+    public GHPullRequestReviewCommentBuilder path(String path) {
+        builder.with("path", path);
+        return this;
+    }
+
+    /**
+     * The position in the diff where you want to add a review comment.
+     *
+     * @param position
+     *            the position
+     * @return the gh pull request review comment builder
+     * @deprecated This parameter is deprecated in GitHub API, use {@link #line(int)} instead.
+     */
+    @Deprecated
+    public GHPullRequestReviewCommentBuilder position(int position) {
+        builder.with("position", position);
```

Comment:
Let's make this internal at most.  If it is not what people should use, we should not let them use it at all.